CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(pennylane_lightning)
+
+# Read and set pennylane_lightning version
+execute_process(COMMAND "/usr/bin/env" "python"  "-c"
+	"with open('${CMAKE_SOURCE_DIR}/pennylane_lightning/_version.py') as f: print(f.readlines()[-1].split()[-1].strip(\"\\\"'\"), end='')"
+	OUTPUT_VARIABLE VERSION_STRING)
+message(STATUS "Pennylane_lightning version ${VERSION_STRING}")
+set(PROJECT_VERSION ${VERSION_STRING})
+
+set(CMAKE_CXX_STANDARD 11) # At least C++11 is required
+find_package(OpenMP REQUIRED) # find OpenMP
+
+if(NOT CMAKE_BUILD_TYPE)
+	set(CMAKE_BUILD_TYPE Release)
+endif()
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=native -W -DVERSION_INFO=${VERSION_STRING}")
+
+
+# Add pybind11
+include(FetchContent)
+FetchContent_Declare(
+	pybind11
+	GIT_REPOSITORY https://github.com/pybind/pybind11.git
+	GIT_TAG        stable 
```

Comment:
I was unaware of the `stable` tag. Nice find! 

Though, I think it may be a better idea to pin to a specific version than a rolling version, to ensure we never hit problems with changes on their side (and we can always bump the version numbers later). For now, I think `v2.6.2` would be good here.